### PR TITLE
Add Intro to Digital Autonomy

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

Added article on digital autonomy,.

## Links to any relevant issues

https://github.com/iotaledger/devx/issues/215

## Type of change

- Documentation 

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: docs/learn/protocols/iota2.0/introduction-to-digital-autonomy.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+# Introduction to Digital Autonomy
+
+Within a myriad of open-source DLTs on the market, cloning an existing solution and changing a few parameters would be very easy – as demonstrated by numerous competitors. But at the IOTA Foundation, we want to build useful technology that solves real problems. Since every other project has some sort of deficiency, this has forced us to build a novel protocol with a radically different architecture that can unlock the full potential of DLT.
```
Comment (jlvandenhout) [inline docs/learn/protocols/iota2.0/introduction-to-digital-autonomy.md:3]: To me this sounds a bit passive aggressive. Could we somehow rephrase this to be more constructive, like explicitly mention the drawbacks of classic DLTs that we want to improve, aka linear vs parallel block production, permissioned access vs fair access, etc.?

Comment (lucas-tortora) [reply docs/learn/protocols/iota2.0/introduction-to-digital-autonomy.md:3]: We need to get many people involved to edit these articles at this stage.
